deployments/common/pip/requirements.txt

```diff
@@ -0,0 +1,14 @@
+numpy==1.20.3
+scipy==1.6.3
+matplotlib==3.4.2
+grpcio==1.37.1
+jupyter==1.0.0
+Cython==0.29.23
+protobuf==3.16.0
+pandas==1.2.4
+healpy==1.14.0
+astropy==4.2.1
+astroquery==0.4.1
+scikit-learn==0.24.2
+hdbscan==0.8.27
+pyvo==1.1
```

Comment:
Can we confirm that this is no longer part of the pull request ?
If merging master back in to this branch resolved it, then why is this still showing up in the  pull request as a change for review ?

Reply:
Looking at the files changed page https://github.com/wfau/aglais/pull/517/files it doesn't look like this file is part of the files changed, so it looks ok to me. Otherwise I'm not sure what to do to have it not appear in the changes.

Reply:
OK, thanks for checking.